servers/image-api/schema.graphql

```diff
@@ -35,7 +35,7 @@ enum ImageFileType {
 Set of parameters that will be used to change an image
 """
 input CachedImageInput {
-  "Id of the image in the returned result set"
+  "ID that will be added to the generated response so you can find it you silly face"
```

Comment:
Perhaps:

ID that will be added to the generated response object so you can find it.  Can be any string that you will use to delineate it when consumed